.cloudbees/workflows/workflow.yaml

```diff
@@ -153,3 +153,12 @@ jobs:
         values-files: |
           charts/example/values.yaml
           charts/example/values-test.yaml
+    - name: install a local helm chart with wait set to false
+      continue-on-error: true
```

Comment:
Why set `continue-on-error=true`?
Isn't the point that It shouldn't fail when `wait=false` and the Deployment doesn't come up initially?

Reply:
Thanks for catching that, my bad, the step does pass with the `wait` flag set to `false`, updated now, please re-review.